site/specs-source/webRtc.json

```diff
@@ -499,7 +499,7 @@
                }
             ],
             "requestBody": {
-               "description": "Subscriptions the participant should be created with",
+               "description": "NOTE: the request body for this endpoint is OPTIONAL and provided as a convenience to avoid calls to multiple endpoints.\nCalling this endpoint with no/empty body will only add the designated participant to the designated session.\n\nSubscriptions the participant should be created with",
```

Comment:
My opinion here...

Include the NOTE: After the description of the request body, not before it. Also indicate that the request body is optional in the description of the request body, i.e.

```
Optional set of subscriptions to set on the participant
```

Might as well be clear about which endpoints we're talking about not having to call, i.e.
```
....provided as a convenience to avoid calls to this and the Update Participant Subscriptions endpoint.
```

I also think we can remove both instances of the word "designated". It should be clear that we're operating on the participant named in the URL path.